config/config/assets/registry-config.yaml.tmpl

```diff
@@ -145,6 +145,31 @@ storage:
     realm: {{ .Spec.Storage.Driver.Azure.BaseURL | quote }}
   {{- end }}
 
+  {{- if .Spec.Storage.Driver.Oss }}
+  oss:
+    accesskeyid: {{ .Spec.Storage.Driver.Oss.AccessKeyID | quote }}
+    bucket: {{ .Spec.Storage.Driver.Oss.Bucket | quote }}
+    region: {{ .Spec.Storage.Driver.Oss.Region | quote }}
+    {{- if .Spec.Storage.Driver.Oss.Endpoint }}
+    endpoint: {{ .Spec.Storage.Driver.Oss.Bucket }}.{{ .Spec.Storage.Driver.Oss.Endpoint }}
```

Comment:
Endpoint needs to contain bucket ?

Reply:
yes, this is a strange behavior defined by aliyun
<img width="990" alt="image" src="https://user-images.githubusercontent.com/188115/205663011-107d2d49-d871-42f2-8daf-d58cb2d0a653.png">

Reply:
also refer https://github.com/goharbor/harbor-helm/blob/185354a01bc00fb9db5943dfd331128fbcd149d8/templates/registry/registry-cm.yaml#L138